dist/redhat-brand-theme.css

```diff
@@ -1046,16 +1046,132 @@
   --pf-v6-c-skeleton--BorderRadius: var(--rh-border-radius-sharp, 0);
 }
 
-.pf-v6-c-modal-box {
-  --pf-v6-c-modal-box--BorderRadius: var(--rh-border-radius-sharp, 0px);
+:root {
+  --rh-color-light-gray: #fafafa;
```

Comment:
The color palette shipping with PF6 is the complete brand color palette. So I’m wondering if - even if they don’t 100% match the old colors, if it’s worth making sure all colors selected with this red hat brand theme are from PF6's shipped color palette? What do you think?